lib/internal/child_process.js

```diff
@@ -510,7 +510,7 @@ function setupChannel(target, channel) {
     if (typeof callback === 'function') {
       process.nextTick(callback, ex);
     } else {
-      this.emit('error', ex);  // FIXME(bnoordhuis) Defer to next tick.
+      process.nextTick(emitErrorNT, this, ex);
```

Comment:
wondering if it's worthwhile to create the separate emitErrorNT function. 

This ought to work:
```
process.nextTick(()=>this.emit('error', ex)); 
```